core/src/main/java/eu/fasten/core/data/Constants.java

```diff
@@ -0,0 +1,17 @@
+package eu.fasten.core.data;
+
+public class Constants {
+
+    public static final String mvnCoordinatePartsJoin = ":";
+
+    public static final String mavenForge = "mvn";
+
+    public static final String opalGenerator = "OPAL";
+
+    public static final int transactionRestartLimit = 3;
+
+    public static final int insertionBatchSize = 4096;
+
+    public static final String mvnRepoEnvVariable = "MVN_REPO";
```

Comment:
Maybe also add FASTEN_DB_PASS env variable constant?

Reply:
Sure, added it along with `PGPASSWORD`